ctapipe/coordinates/angular_frames.py

```diff
@@ -85,26 +84,56 @@ class NominalFrame(BaseCoordinateFrame):
 
     Frame attributes:
 
-    * ``pointing_direction``
+    * ``array_direction``
       Alt,Az direction of the array pointing
 
     The Following attributes are carried over from the telescope frame
     to allow a direct transformation from the camera frame
 
     * ``focal_length``
-      Focal length of the telescope 
+      Focal length of the telescope
     * ``rotation``
       Rotation angle of the camera (0 in most cases) [deg]
     * ``pointing_direction``
       Alt,Az direction of the telescope pointing
 
     """
-    default_representation = CartesianRepresentation
+
+    default_representation = PlanarRepresentation
+    pointing_direction = FrameAttribute(default=None)
     array_direction = FrameAttribute(default=None)
+
+
+class HorizonFrame(BaseCoordinateFrame):
+    """
+    Horizon coordinate frame. Spherical system used to describe the direction
+    of a given position, in terms of the altitude and azimuth of the system. In
+    practice this is functionally identical as the astropy AltAz system, but this
+    implementation allows us to pass array pointing information, allowing us to directly
+    transform to the Horizon Frame from the Camera system.
+
+    The Following attributes are carried over from the telescope frame
+    to allow a direct transformation from the camera frame
+
+    * ``array_direction``
+      Alt,Az direction of the array pointing
+    * ``pointing_direction``
+      Alt,Az direction of the telescope pointing
```

Comment:
Hi, hmm OK that makes more sense as it is quite consistent with the documentation parameters. But looking at the astropy code they don't seen to follow this scheme, e.g.

http://docs.astropy.org/en/stable/_modules/astropy/coordinates/builtin_frames/altaz.html#AltAz